project-koku/koku
# COST-704: Adding current_month_data flag to Sources API response

Adding `current_month_data` flag to Sources API response.  This boolean will be true if we successfully processed data for the source for the current month.  @dlabrecq will be able to use this to provide a better empty state user experience in the UI.

**Testing**
1. Add AWS-local account with nise data for only previous month.  Verify that sources current_month_data=False
2. Update nise yaml to include data for current month and re-generate data.  Verify flag is true

**Test Results**
[source_data_ready_ut.txt](https://github.com/project-koku/koku/files/5538026/source_data_ready_ut.txt)
